Use backticks to render URLs etc as monospaced (#159)
AMWA-TV/nmos-discovery-registration: docs/2.0. APIs.md

All NMOS APIs MUST use a path in the following format. Other HTTP resources MAY be presented on the same port by the Node, provided all NMOS resources are available from the /x-nmos/ path as follows:

```
http(s)://<ip address or hostname>:<port>/x-nmos/<api type>/<api version>/
```

At each level of the API from the base resource upwards, the response SHOULD include a JSON format array of the child resources available from that point.

> API clients using DNS-SD for API discovery SHOULD construct these paths by making use of the TXT records `api_proto` and `api_ver`, along with addresses and ports resolved via DNS-SD. API clients which are discovering Node APIs via a Query API SHOULD construct Node API paths using the corresponding data available within the `api` attributes within the Query API `/nodes/` path.

Reviewer: ```suggestion
API clients using DNS-SD for API discovery SHOULD construct these paths by making use of the TXT records `api_proto` and `api_ver`, along with addresses and ports resolved via DNS-SD. API clients which are discovering Node APIs via a Query API SHOULD construct Node API paths using the corresponding data available within the `api` attributes within the Query API `/nodes` resources.
```

> API clients using DNS-SD for API discovery SHOULD construct these paths by making use of the TXT records 'api\_proto' and 'api\_ver', along with addresses and ports resolved via DNS-SD. API clients which are discovering Node APIs via a Query API SHOULD construct Node API paths using the corresponding data available within the `api` attributes within the Query API `/nodes/` path.

Reviewer: Did you decide not these two for some reason? ...Also, what about TXT?

```suggestion
API clients using DNS-SD for API discovery SHOULD construct these paths by making use of the TXT records `api_proto` and `api_ver`, along with addresses and ports resolved via DNS-SD. API clients which are discovering Node APIs via a Query API SHOULD construct Node API paths using the corresponding data available within the `api` attributes within the Query API `/nodes/` path.
```

> Where protocol and versioning data is not available, clients MAY assume a v1.0 API, which operates via the `http` protocol only.

Reviewer: Or maybe just this?

```suggestion
Where protocol and versioning data is not available, clients MAY assume a v1.0 API, which operates via HTTP only.
```

]
```

- Appending `/v1.0/` to the API base resource will request version 1.0 of the API if available.
- The versioning format is `v<#MAJOR>.<#MINOR>`
- MINOR increments will be performed for non-breaking changes (such as the addition of attributes in a response)
- MAJOR increments will be performed for breaking changes (such as the renaming of a resource or attribute)

> - Versions MUST be represented as complete strings. Parsing MUST proceed as follows: separate into two strings, using the point (.) as a delimiter. Compare integer representations of MAJOR, MINOR version (such that v1.12 is greater than v1.5).

Reviewer: Does this meet the (unwritten) rule or not?

Also why are MAJOR and MINOR capitalized here and not backticked? And why are the words MAJOR and MINOR preceded by a `#` above?

```suggestion
- Versions MUST be represented as complete strings. Parsing MUST proceed as follows: separate into two strings, using the point (`.`) as a delimiter. Compare integer representations of MAJOR, MINOR version (such that v1.12 is greater than v1.5).
```
